cg/cli/archive.py

```diff
@@ -82,3 +82,48 @@ def delete_file(context: CGConfig, dry_run: bool, file_path: str):
         data_flow_config=context.data_flow,
     )
     spring_archive_api.delete_file(file_path=file_path, dry_run=dry_run)
+
+
+@archive.group("retrieve-spring")
```

Comment:
This was just my working title. The whole command will be quite long:
`cg archive retrieve-spring case justhusky`
Do you have any suggestions?

Reply:
Maybe just `retrieve` then?